main.css

```diff
@@ -32,9 +31,10 @@ width: 100%;
 
 
 nav{
-    
+    position: relative;
+    top: 230px;
     width: 62%;
-    height: 5%;
+    height: 30px;
     margin: 60px;
```

Comment:
If you remove this margin you would see how it affects the page display. You have to be careful with margins